modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -32,11 +32,6 @@ The following Technology Preview features are enabled by this feature set:
 ** Admin Network Policy and Baseline Admin Network Policy. Enables `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources, which are part of the Network Policy V2 API, in clusters running the OVN-Kubernetes CNI plugin. Cluster administrators can apply cluster-scoped policies and safeguards for an entire cluster before namespaces are created. Network administrators can secure clusters by enforcing network traffic controls that cannot be overridden by users. Network administrators can enforce optional baseline network traffic controls that can be overridden by users in the cluster, if necessary. Currently, these APIs support only expressing policies for intra-cluster traffic. (`AdminNetworkPolicy`)
 ** `MatchConditions` is a list of conditions that must be met for a request to be sent to this webhook. Match conditions filter requests that have already been matched by the rules, namespaceSelector, and objectSelector. An empty list of `matchConditions` matches all requests. (`admissionWebhookMatchConditions`)
 ** Gateway API. To enable the {product-title} Gateway API, set the value of the `enabled` field to `true` in the `techPreview.gatewayAPI` specification of the `ServiceMeshControlPlane` resource.(`gateGatewayAPI`)
-** `sigstoreImageVerification`
-** `gcpLabelsTags`
-** `vSphereStaticIPs`
-** `routeExternalCertificate`
-** `automatedEtcdBackup`
```

Comment:
does this fall under the scope of this PR?

Reply:
No, sorry, you can ignore this. I'll be exporting the content to a KB article anyway when approved.